pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,21 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+The programmatically determined link context is a list of elements. All elements in the list must be [included in the accessibility tree][], and have one of the following relationships to the link:
```

Comment:
There is still an issue on how many elements matching the condition are in the Programmatically Determined Link Context, making the definition ambiguous.

```html
<span id="label>Foo 1</span>
<p>Foo 2
<ul>
<li>Foo 3
<table>
<tr><th>Foo 4</th></tr>
<tr><td>Foo 5 <a href="#" aria-described-by="label">Bar</a></td></tr>
</table>
</ul>
</p>
```
What is the programmatically determined link context of this link?
With the current definition, any subset of {Foo 1, Foo 2, Foo 3, Foo 4, Foo 5} (including the empty set!) can be.

---
Also,  "list" kinda suggest ordering of the elements in the context, which is probably not what you want. "set" seems better.

---
```suggestion
The programmatically determined link context is the set of all elements that are [included in the accessibility tree][], and have one of the following relationships to the link:
```